_data/examples.json

```diff
@@ -196,6 +196,10 @@
         "title": "Table Heatmap",
         "style": "background-size: auto"
       },
+      {
+        "name": "rect_heatmap_weather",
+        "title": "Table Heatmap of Weather"
```

Comment:
I wonder if we should call this "Annual Weather Heatmap"?

(We can keep file name the same.)

Reply:
That sounds good to me. I can go ahead and update the title.